internal/core/command/types.go

```diff
@@ -29,14 +30,19 @@ type serviceCommand struct {
 	contract.Device
 	internal.HttpCaller
 	*http.Request
+	loggingClient logger.LoggingClient
```

Comment:
Alternate approach to simply adding field to receiver is to create a factory function and replace all existing non-error struct literal definitions with a call to the factory function.  [Boy Scout Rule](http://pragmaticcraftsman.com/2011/03/the-boy-scout-rule/).  Ensures single consistent mechanism for instantiating receiver (which ensures all fields in receiver are always properly initialized).